staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1746,6 +1749,42 @@ type CSIVolumeSource struct {
 	NodePublishSecretRef *LocalObjectReference `json:"nodePublishSecretRef,omitempty" protobuf:"bytes,5,opt,name=nodePublishSecretRef"`
 }
 
+// Represents ephemeral storage that is handled by a normal storage driver.
+type EphemeralVolumeSource struct {
+	// Will be created as a stand-alone PVC to provision the volume.
+	//
+	// The pod in which this EphemeralVolumeSource is embedded will be the
+	// owner, i.e. the PVC will be deleted together with the pod.
+	// The name of the PVC will be `<pod name>-<volume name>` where `<volume name>`
+	// is the name from the `PodSpec.Volumes` array entry. Care must
+	// be taken to keep this concatendated name shorter enough that it
+	// can be used as object name.
+	//
```

Comment:
Describe what happens when a user creates the PVC up front but pod isn't listed as an owner.  Does pod fail to schedule, fail to admit, fail to start?  Why or why not would someone want adoption to work?  Could a higher level controller like statefulset controller use this?  Does statefulset conrtoller have to verify that no naming conflict exists with the pod template?

If a mutating admission plugin creates an ephemeral volume on the fly at pod creation, what happens?  I.e. ephemeral volumes were slated for admission style PodPreset / magic injection of secrets (give every pod in this namespace a kerberos identity injected as an ephemeral volume), how would that that admission plugin interoperate with the proposed ephemeral creating controller?